src/Our.Umbraco.Vorto/Extensions/IPublishedContentExtensions.cs

```diff
@@ -85,7 +85,10 @@ public static class IPublishedContentExtensions
             string cultureName = null, bool recursive = false,
             string fallbackCultureName = null)
         {
-            var hasValue = content.DoHasVortoValue(propertyAlias, cultureName, recursive);
+			if (fallbackCultureName.IsNullOrWhiteSpace())
+				fallbackCultureName = Vorto.DefaultFallbackCutltureName;
```

Comment:
Hi Matt, you have a typo in your method name:
DefaultFallbackCutltureName => DefaultFallbackCultureName

Reply:
Typical me :)